packages/benchmark/doc/benchmarkerator.md

```diff
@@ -157,13 +161,21 @@ Timing results and other collected metrics are output to _stdout_.  Two batches
 of information are provided: one for the setup phase and one for the benchmark
 rounds themselves (in aggregate).
 
-If you specify the `--dump FILEPATH` command line option, a JSON-formatted
+If you specify the `--output FILEPATH` command line option, a JSON-formatted
 (i.e., machine readable) version of this same data will be output to the
 indicated file.
 
 Output results include execution times (according to Node's nanosecond clock),
 crank counts, and the various kernel resource usage data reported by
 `controller.getStats()`.
 
+Per-vat JS engine-level profiling is enabled for a given vat via the `--profile
+VATID` command line option.  Note that this option is only available when the
+vat manager type is `node-process` or `xs-worker`.  For `node-process` vats, the
+profiling data will be placed in the file `CPU.${vatID}:${vatName}.cpuprofile`.
```

Comment:
I think the issue you point out with `:` in file names is legit, but the colon is inherited from the name tag that the vat worker process launcher puts on the subprocess command line, so changing it would entail changing _that_.  I don't _think_ we have any tooling that's depending on that format for the name, but I'm not 100% certain either.

VatIDs take the form `v${n}`.

Reply:
I'll convert the colon to a dash.  However the `CPU` part is how v8 distinguishes CPU profiles from heap profiles, so I think I want to leave that bit unchanged.